Azure/azure-cli
# [Keyvault] BREAKING CHANGE: Fix #13752: az keyvault create not idempotent. Creating existing keyvault will fail.

**Description**

Fix #13752 
CLI will set default access_policy when running `az keyvault create`. That will overwrite existing settings if the given vault already exists.
After discussion with customer and keyvault server side engineer, we decided to keep align with what powershell does in such situation: Throw exception if the given `vault_name` exists.

**Testing Guide**

```
> az keyvault create -n existingvault -g testrg
The specified vault: existingvault already exists
```
**Notes:** All module tests containing `keyvault create` need to be re-recorded or changed to `KeyVaultPrepare`

---

This checklist is used to make sure that common guidelines for a pull request are followed.
